pages/common/chmod.md

```diff
@@ -1,36 +1,24 @@
 # chmod
 
-> Change the access permissions of a file or directory.
+> Change file/directory access permissions.
 > More information: <https://www.gnu.org/software/coreutils/chmod>.
 
-- Give the [u]ser who owns a file the right to e[x]ecute it:
+- Add [r]ead/[w]rite/e[x]ecute permissions for a file/directory owner:
 
-`chmod u+x {{file}}`
+`chmod +{{r|w|x}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Give the [u]ser rights to [r]ead and [w]rite to a file/directory:
+- Add specific permissions for an owner/users in [g]roup/[o]ther users/[a]ll at once:
 
-`chmod u+rw {{file_or_directory}}`
+`chmod {{u|g|o|a}}+{{r}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Remove e[x]ecutable rights from the [g]roup:
+- Add/remove/replace specific permissions:
 
-`chmod g-x {{file}}`
+`chmod {{u}}{{+|-|=}}{{r}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Give [a]ll users rights to [r]ead and e[x]ecute:
+- Remove all permissions:
 
-`chmod a+rx {{file}}`
+`chmod {{u}}= {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

Comment:
When would you ever want to execute this? I recommend at least changing `u` to `o`...

Reply:
Yes, thanks. 👍🏻 Good idea.